src/bp-templates/bp-nouveau/buddypress/members/single/groups.php

```diff
@@ -10,7 +10,7 @@
 <nav class="<?php bp_nouveau_single_item_subnav_classes(); ?>" id="subnav" role="navigation" aria-label="<?php esc_attr_e( 'Groups menu', 'buddypress' ); ?>">
 	<ul class="subnav">
 
-		<?php if ( bp_is_my_profile() ) : ?>
+		<?php if ( bp_core_can_edit_settings() ) : ?>
```

Comment:
We can simply remove this check actually and simply display the Member's Groups subnav. The Invitations Tab will only show for the users seeing their own profile or for Admins. As you noted into your comment on Track, the Invitations subnav will only show if `bp_core_can_edit_settings()`.

Reply:
Good point, thank you.